app/views/users/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<%# <h1>User Profile</h1> %>
+<div class="user-navigation">
+  <a href="<%= users_path %>">
+    <button class="back-btn">⬅</button>
+  </a>
+</div>
+<section class="user-profile-page">
+  <div class="user-profile">
+    <% if @user.photo.present? %>
+      <%= image_tag @user.photo, alt: @user.name, class: 'user-thumbnail' %>
+    <% else %>
+      <span class="user-thumbnail placeholder">No Photo</span>
+    <% end %>
+    <p><%= @user.name %></p>
+    <p>Number of Posts: <%= @user.posts.count %></p>
+  </div>
+  <div class="user-bio">
+    <h3>Bio:</h3>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <div class="user-posts">
+    <% if @user.posts.any? %>
+      <ul class="user-posts-info">
+        <% @user.posts.each do |post| %>
```

Comment:
- [ ] Based on the requirements, we are supposed to displayed only the 3 most recent posts from the user. For this specify, you're required to kindly call the `#recent_posts` method you got in the User model on user. Let's kindly adhere to the project's requirements once more :+1: and fix :wrench: this issue.